src/main/kotlin/uk/gov/dwp/dataworks/controllers/ConnectionController.kt

```diff
@@ -77,16 +87,17 @@ class ConnectionController {
     }
 
     fun handleRequest(userName: String, requestBody: DeployRequest):String {
-        if (existingUserServiceCheck.check(userName, configurationResolver.getStringConfig(ConfigKey.ECS_CLUSTER_NAME))){
+        if (activeUserTasks.contains(userName)) {
             logger.info("Redirecting user to running containers, as they exist")
         } else {
-            taskDeploymentService.runContainers(
+            val userTask = taskDeploymentService.runContainers(
```

Comment:
Can this fail? I note the TaskDestroyService looks for exceptions, but the TaskCreateService doesn't seem to do any checking, so could we be adding an userTask when deployment failed?

Reply:
If there were any failures in the request then it would drop out of the `AwsCommunicator` before it returns a value, therefore we don't need to do any checking here